cms/blog/templates/cms_blog/blog_categories.html

```diff
@@ -0,0 +1,11 @@
+<div class="blog-category">
+    {% if categories %}
+        {% for category in categories %}
+            {% if forloop.first %}
+                {{ category.translated_name|upper }}
+            {% else %}
+                &nbsp; | &nbsp; {{ category.translated_name|upper }}
```

Comment:
Those htmlentities are proably not needed or? In any case django has a builtin template filter [join](https://docs.djangoproject.com/es/1.10/ref/templates/builtins/#join), that could replace the for loop.